packages/connect-explorer-nextra/src/pages/index.mdx

```diff
@@ -1,10 +1,15 @@
+import { Card } from '@trezor/components';
+
 # Trezor Connect SDK
 
+<Card>
```

Comment:
Potřebuju promyslet jak to do těch markdown souborů zavedeme, manuálně to dělat nechci 😃